### DISCUSSION TITLE
How to associate a tenant ID

### DISCUSSION THREAD

**Discussion by akshatknsl** [Kong/kong#9193] [Help]
We have a scenario where we have multiple consumers belonging to a common tenant, and after authentication, this tenant information needs to be available to the backend.

How can this be done using kong?

**Comment by chronolaw** [comment]
Maybe you can use different `username` but the same `custom_id`. Refer to https://docs.konghq.com/gateway/latest/admin-api/#consumer-object .

**Reply by hanshuebner** [reply]
@akshatknsl Is your problem solved?
